udata/core/reuse/models.py

```diff
@@ -40,12 +41,14 @@ def hidden(self):
                     db.Q(datasets__0__exists=False) |
                     db.Q(deleted__ne=None))
 
-
+decorator = mdstrip_field('description')
+@decorator
```

Comment:
We need to figure out why, what is the error? I guess it's because you're applying it to a class and as such the first parameter of your decorator should be the class instance (even if not in use).